tests/tester.nim

```diff
@@ -1,33 +1,33 @@
 # Copyright (C) Dominik Picheta. All rights reserved.
 # BSD License. Look at license.txt for more info.
 
-import testscommon
+# import testscommon
 
 # suits imports
 
-import tinitcommand
-import tcheckcommand
-import tcleancommand
-import tdevelopfeature
-import tissues
-import tlocaldeps
-import tlockfile
-import tmisctests
-import tmoduletests
-import tmultipkgs
-import tnimbledump
-import tnimblerefresh
-import tnimbletasks
-import tnimscript
-import tshellenv
-import tpathcommand
-import treversedeps
-import truncommand
+# import tinitcommand
+# import tcheckcommand
+# import tcleancommand
+# import tdevelopfeature
+# import tissues
+# import tlocaldeps
+# import tlockfile
+# import tmisctests
+# import tmoduletests
+# import tmultipkgs
+# import tnimbledump
+# import tnimblerefresh
+# import tnimbletasks
+# import tnimscript
+# import tshellenv
+# import tpathcommand
+# import treversedeps
+# import truncommand
 import tsetupcommand
-import ttestcommand
-import ttwobinaryversions
-import tuninstall
-import ttaskdeps
+# import ttestcommand
+# import ttwobinaryversions
+# import tuninstall
+# import ttaskdeps
```

Comment:
Why were these tests disabled?

Reply:
Apologies, I committed the file by mistake. Reverted it.